requirements.txt

```diff
@@ -1,2 +1,4 @@
 requests==2.9.1
 nose==1.3.7
+wextractor==0.1.dev3
```

Comment:
I think we should drop this as a dependency and replace whatever functionality is needed